iib/workers/tasks/build.py

```diff
@@ -971,7 +985,7 @@ def handle_add_request(
         )
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
-    output_pull_spec = _create_and_push_manifest_list(request_id, arches)
+    output_pull_spec = _create_and_push_manifest_list(request_id, arches, build_tags)
```

Comment:
Is it singular or plural? Do we get the one pull spec or more? I guess one but better to ask